freestyle-logging/jvm/src/main/scala/io/freestyle/loggingJVM.scala

```diff
@@ -0,0 +1,32 @@
+package io.freestyle
+
+import cats.MonadError
+import io.freestyle.logging._
+import journal._
+
+object loggingJVM {
+
+  object implicits {
+    implicit def freeStyleLoggingInterpreter[M[_], C: Manifest](implicit ME: MonadError[M, Throwable]): LoggingM.Interpreter[M] =
+      new LoggingM.Interpreter[M] {
+
+        val logger = Logger[C]
+
+        def debugImpl(msg: String): M[Unit] = ME.catchNonFatal(logger.debug(msg))
```

Comment:
@raulraja Maybe I'm wrong but the return type of all the Logger methods is `Unit`. If we want to integrate the logging stuff into a program monadic flow, we should lift the return value into the applicative functor.

Initially I was using `Monad.pure` but I changed to `MonadError.catchNonFatal` after talking to you.